1.preprocessing-data/1.illumcorrect-data.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+jupyter nbconvert --to python illumcorrect-data.ipynb
+python illumcorrect-data.py
```

Comment:
```suggestion
python illumcorrect-data.py

```

New linesss EOF ahhhh

Reply:
I have that parameter turned on UGHHHH

Reply:
Nevermind it got turned off somehow :(